src/distributions/semeru/installer.ts

```diff
@@ -0,0 +1,168 @@
+import { JavaBase } from '../base-installer';
+import { JavaDownloadRelease, JavaInstallerOptions, JavaInstallerResults } from '../base-models';
+import semver from 'semver';
+import { extractJdkFile, getDownloadArchiveExtension, isVersionSatisfies } from '../../util';
+import * as core from '@actions/core';
+import * as tc from '@actions/tool-cache';
+import fs from 'fs';
+import path from 'path';
+import { ISemeruAvailableVersions } from './models';
+
+export class SemeruDistribution extends JavaBase {
+  constructor(installerOptions: JavaInstallerOptions) {
+    super('IBM_Semeru', installerOptions);
+  }
+
+  protected async findPackageForDownload(version: string): Promise<JavaDownloadRelease> {
+    if (
+      this.architecture !== 'x64' &&
+      this.architecture !== 'x86' &&
+      this.architecture !== 'ppc64le' &&
+      this.architecture !== 'ppc64' &&
+      this.architecture !== 's390x' &&
+      this.architecture !== 'aarch64'
```

Comment:
Could you please change it to the array of supportedArchitectures and use includes method. Do we need such checks ? Could we rely on API only ?

Reply:
Hopefully, depends on when @AdamBrousseau has time on their hands